padertorch/contrib/examples/speech_enhancement/mask_estimator/README.md

```diff
@@ -0,0 +1,57 @@
+Simple Mask Estimator
+=============
+
+This directory contains scripts to train and evaluate a simple mask estimator
+inspired by [1].
+
+Results
+-------
+
+The mask estimator model achieves the following results on
+the CHiME 3 simulated evaluation set:
+
+
+data type         |  pesq         | stoi         |      sdr 
+:------------------|--------------|--------------|--------------:
+observed           |  1.07       |  0.672       |  -0.79 dB
+masked             |  1.22       |  0.736       |  5.68 dB
+beamformed         |  1.91       |  0.958       |   17.10 dB  
+
+Masked and observed are evaluated on the first channel of the 6ch track.
+
+Training
+--------
+
+A storage root must be set with `export STORAGE_ROOT=/path/to/your/storage`.
+After installing `padertorch`, a training can for example be started with
+
+```bash
+$ STORAGE_ROOT=/path/to/your/storage; python -m padertorch.contrib.examples.speech_enhancement.simple_mask_estimator.train with database_json=/path/to/json
+```
```

Comment:
I guess the JSON needs a certain structure (e.g., certain keys) to work with the example. Is it possible to give a short description of the JSON's structure?

Reply:
done